torch/_inductor/cpu_vec_isa.py

```diff
@@ -337,9 +333,10 @@ def valid_vec_isa_list() -> List[VecISA]:
                         if re.search(r"[\^ ]+vxe[\$ ]+", group):
                             isa_list.append(VecZVECTOR())
                             break
+
     elif arch == "ppc64le":
         isa_list.append(VecVSX())
-    elif arch == "aarch64":
+    elif arch in ["aarch64", "arm64"]:
```

Comment:
```suggestion
    elif arch == "aarch64":
```

Reply:
"arm64" is for keeping vectorization for MacOS